source/03-components/button/button.twig

```diff
@@ -50,13 +58,13 @@
   {% endset %}
 
   {% if is_button_tag %}
-    <button {{ add_attributes({ 'class': classes }) }}{% if is_disabled %} disabled{% endif %}{% if button_type %} type="{{ button_type }}"{% endif %}>{{ button_content }}</button>
+    <button {{ add_attributes(button_attributes) }}{% if is_disabled %} disabled{% endif %}{% if button_type %} type="{{ button_type }}"{% endif %}>{{ button_content }}</button>
   {% else %}
-    <a {{ add_attributes({ 'class': classes }) }} href="{{ url }}"{% if no_follow %} rel="nofollow"{% endif %}{% if storybook %} target="_self"{% endif %}>{{ button_content }}</a>
+    <a {{ add_attributes(button_attributes) }} href="{{ url }}"{% if no_follow %} rel="nofollow"{% endif %}{% if storybook %} target="_self"{% endif %}>{{ button_content }}</a>
   {% endif %}
 
   {% if is_demo %}
-    <a {{ add_attributes({ 'class': classes }) }} href="{{ url }}"{% if no_follow %} rel="nofollow"{% endif %}{% if storybook %} target="_self"{% endif %}>{{ demo_button_content }}</a>
-    <button {{ add_attributes({ 'class': classes }) }} disabled>{{ disabled_button_content }}</button>
+    <a {{ add_attributes(button_attributes) }} href="{{ url }}"{% if no_follow %} rel="nofollow"{% endif %}{% if data_attributes %} {{ data_attributes }}{% endif %}{% if storybook %} target="_self"{% endif %}>{{ demo_button_content }}</a>
```

Comment:
Is this data_attributes bit left over from a previous approach?

Reply:
Yep, just removed it.